content/en/actions/agents/_index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Agents
+description: Build custom AI agents that can access the Action Catalog. 
+disable_toc: false
+further_reading:
+- link: "/actions/actions_catalog/"
+  tag: "Documentation"
+  text: "Action Catalog"
+- link: "https://www.datadoghq.com/knowledge-center/aiops/ai-agents/"
+  tag: "Knowledge Center"
+  text: "What are AI agents and how do they work?"
+---
+
+{{< callout url="https://app.datadoghq.com/actions/agents" btn_hidden="false" header="Join the Preview!" >}}
+Agents is in Preview. Click <b>Request Access</b> and fill in the Datadog Product Preview Program form.
+{{< /callout >}}
+
+## Overview
+
+You can build custom AI agents to act on your observability data and third-party integrations. Your agents can use any action from the [Action Catalog][7] to investigate problems, start remediation, or manage resources.
+
+When building an agent, you can choose a template to get started. Templates come with pre-filled instructions and relevant tools. Templates are named after their main role, like Security Analyst, Log Analyzer, Incident Responder, or DevOps Assistant. 
+
+{{< img src="/actions/agents/blank-new-agent.png" alt="The Agents page, filtered to show only 'My agents'" style="width:100%;" >}}
+
+## The Agents dashboard
+
+You can create a new agent from the [Agents][1] page. The page lists information about existing agents, including the following:
+- Agent Name
+- Description
+- Author
+- Creation date
+
+Hover over an agent for options to clone or delete it. You can also enable the **My agents** toggle to see only agents that you've created:
+
+{{< img src="/actions/agents/agent-dashboard.png" alt="The Agents page, filtered to show only 'My agents'" style="width:100%;" >}}
+
+## Create an agent
+
+### Build an agent from a blueprint
+
+Blueprints are helpful starter agents that cover common use cases. They come loaded with instructions for the agent's behavior and helpful tools from the Action Catalog. Blueprints also showcase best practices for setting up agent functionality.
+
+To build an agent from a blueprint:
+
+1. Navigate to the [Agents][1] page. 
+1. In **Agent Blueprints**, click the blueprint you want to use. 
+    1. Alternatively, you can click **New Agent** and browse the full list of blueprints.
```

Comment:
I'd make this one step, since the second line is basically the same thing, just an alternative way of doing it. Proper numbered/lettered substeps are better when you have a longer sub-process with multiple steps.

```suggestion
1. In **Agent Blueprints**, click the blueprint you want to use. Alternatively, you can click **New Agent** and browse the full list of blueprints.
```